libraries/standard/http/include/http_client.h

```diff
@@ -381,20 +399,31 @@ HTTPStatus_t HTTPClient_InitializeRequestHeaders( HTTPRequestHeaders_t * pReques
  * Upon return, pRequestHeaders->headersLen will be updated with the number of
  * bytes written.
  *
- * TODO: Expand documentation.
+ * The maximum value for valueLen and nameLen is ( UINT32_MAX >> 2 ) and also
+ * cannot be set to 0. Otherwise, #HTTP_INVALID_PARAMETER is returned.
+ *
+ * If the following header fields are set by this method,
```

Comment:
This block needs to be deleted per recent change to decouple with HTTPClient_InitializeRequestHeaders().